release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1621,6 +1621,10 @@ sourceStrategy:
 [id="ocp-4-12-image-registry-bug-fixes"]
 ==== Image Registry
 
+* Previously, the Image Registry Operator did not have a `progressing` condition for the `node-ca` daemon set and used `generation` from an incorrect object. Consequently, the `node-ca` daemon set could be marked as `degraded` while the Operator was still running. This update adds the `progressing` condition, which indicates that the installation is not complete. As a result, the Image Registry Operator successfully installs the `node-ca` daemon set and the installer waits until it is fully deployed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2093440[(*BZ#2093440*)])
```

Comment:
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2093440[(*BZ#2093440*)])
^ this BZ link has sneaky double parentheses